src/components/AccountPickerItem.jsx

```diff
@@ -12,61 +12,99 @@ import Spinner from 'cozy-ui/react/Spinner'
 import { getAccount } from '../ducks/accounts'
 import { getAccountLogin, getAccountName } from '../lib/helpers'
 import { getErrorTitle } from '../lib/konnectors'
+import { getKonnectorsInMaintenance } from '../reducers'
 
-export const AccountPickerItem = ({
-  t,
-  connection,
-  hasUpdate,
-  konnectorSlug,
-  account
-}) => {
-  const { error, hasError, isRunning, isConnected } = connection
-  const accountName = getAccountName(account)
-  const accountLogin = getAccountLogin(account)
-  const nameAndLoginDiffer = accountName !== accountLogin
-  return (
-    <NavLink
-      to={`/connected/${konnectorSlug}/accounts/${account._id}`}
-      className={classNames(
-        styles['col-account-picker-button'],
-        styles['col-account-picker-button-account']
-      )}
-    >
-      <div className={styles['col-account-picker-button-infos']}>
-        <span>{accountName}</span>
-        {nameAndLoginDiffer && <small>{accountLogin}</small>}
-      </div>
-      {hasError && (
-        <div
-          className={classNames(
-            styles['col-account-picker-button-status'],
-            styles['col-account-picker-button-status--error']
-          )}
-        >
-          <span>
-            {getErrorTitle(t, error, key => `connection.error.${key}.title`)}
-          </span>
-          <Icon icon="warning" />
-        </div>
-      )}
-      {hasUpdate &&
-        !hasError && (
+const MAINTENANCE = 'MAINTENANCE'
+const ERRORED = 'ERRORED'
+const HAS_UPDATE = 'HAS_UPDATE'
+const CONNECTED = 'CONNECTED'
+
+const getStatus = ({ hasError, hasUpdate, isConnected, inMaintenance }) => {
+  if (inMaintenance && hasUpdate) return HAS_UPDATE
+  if (inMaintenance) return MAINTENANCE
+  if (hasError) return ERRORED
+  if (hasUpdate) return HAS_UPDATE
+  if (isConnected) return CONNECTED
+}
+
+export class AccountPickerItem extends PureComponent {
+  getStatusContent = () => {
+    const { t, inMaintenance, connection, hasUpdate } = this.props
+    const { error, hasError, isConnected } = connection
+
+    const the_status = getStatus({
```

Comment:
We only use camelCase for JS variable naming, and I would prefer just `status` or `accountStatus`

Reply:
In my experience, `the` prefix is often a code smell.